build.go

```diff
@@ -151,8 +151,8 @@ func Build(build BuildFunc, options ...Option) {
 	logger.Debugf("Buildpack: %+v", ctx.Buildpack)
 
 	API := strings.TrimSpace(ctx.Buildpack.API)
-	if API != "0.5" && API != "0.6" {
-		config.exitHandler.Error(errors.New("this version of libcnb is only compatible with buildpack APIs 0.5 and 0.6"))
+	if API != "0.5" && API != "0.6" && API != "0.7" {
```

Comment:
Do we want to meet libcnb v2 to just api 0.7 and greater?

Reply:
Yes, I think we should talk more about that. My $0.02, I'm torn. Part of me thinks it would be good to start fresh and remove some of the cruft from the older APIs. Part of me thinks the overhead is small and why not support more APIs?

Definitely curious to see what others think. I'll raise this up in the next BAT meeting.